OpenRA.Mods.Common/Effects/Beacon.cs

```diff
@@ -48,6 +44,12 @@ public class Beacon : IEffect, IScriptBindable, IEffectAboveShroud
 			this.duration = duration;
 			this.delay = delay;
 
+			if (!string.IsNullOrEmpty(beaconSequence))
```

Comment:
It looks like this is identical in all but name to `circleSprite`, so can we rename that instead of adding a new animation?  Should also rename `arrowSprite` to `arrowSequence` for consistency.

Reply:
This uses `PlayRepeating` while `circleSequence` only plays once, so I don't know what you mean.

Reply:
> 11:55:10 (abcdefg30) is https://github.com/OpenRA/OpenRA/pull/16006#discussion_r263998433 resolved?
11:58:10 (pchote) it still feels gross to me to have two distinct fields that do almost the same thing, with hardcoded naming that doesn't actually mean much
11:58:26 (pchote) but thats a bigger problem than the pr can deal with
11:58:35 (pchote) so yeah, consider it resolved
11:58:58 (abcdefg30) ok